cclib/parser/molcasparser.py

```diff
@@ -237,6 +241,14 @@ def extract(self, inputfile, line):
                 target = float(line.split()[-1])
                 scftargets.append(target)
 
+            self.skip_lines(inputfile,['Threshold for linear dependence', 'Threshold at which DIIS is turned on', \
+                                         'Threshold at which QNR/C2DIIS is turned on'])
+            line = next(inputfile)
+
+            if line[6:31] == 'Threshold for Norm(delta)':
+                target = float(line.split()[-1])
+                scftargets.append(target)
```

Comment:
Is there a unit test that you can update to reflect the changes in `scftargets` and `scfvalues`?

Reply:
I don't think that there is a test that would reflect this change. We were earlier parsing 3 targets (or thresholds)
```
scftargets = [Threshold for SCF energy change, Threshold for density matrix, \
Threshold for Fock matrix]
```
And similarly we were parsing the respective `scfvalues`.

Now we are similarly parsing 4 such `scftargets` and respective `scfvalues`.

There are tests that checks for dimensions of `geotargets` and `geovalues`,
```python
    def testscftargetdim(self):
        """Do the scf targets have the right dimensions?"""
        self.assertEquals(self.data.scftargets.shape, (len(self.data.scfvalues), len(self.data.scfvalues[0][0]))) # equals 4
```
But there is no test that actually checks the `scftargets` and `scfvalues` since they differ for each software.

Reply:
@langner Is there a problem then? I don't understand.

Reply:
No, no problem. I am just saying we don't have a test to actually check the values of `scftargets` and `scfvalues` since they are different for each parser and even for each job (Since `scftargets` can be specified by the user).

Reply:
@langner Does #545 satisfy your concern about the unit test here?

Reply:
Yes, I think the new tests in #545 are the way to go. Thanks!